modules/unsupported/stac-store/src/main/java/org/geotools/stac/store/STACDataStoreFactory.java

```diff
@@ -72,6 +72,15 @@ public class STACDataStoreFactory implements DataStoreFactorySpi {
                     1000,
                     Collections.singletonMap(Parameter.MIN, 1));
 
+    public static final Param HARD_LIMIT =
+            new Param(
+                    "hardLimit",
+                    Integer.class,
+                    "How many items to fetch from a collection, tops, even while paging. Set to zero or negative to disable.",
+                    false,
+                    100000,
```

Comment:
can't we refer to the static variable here?

Reply:
You mean STACDataStore.DEFAULT_HARD_LIMIT? Oh yeah, it's actually possible, good point.

Reply:
Fixed